tests/test_core/test_new_alignment.py

```diff
@@ -9,6 +9,9 @@
     AlignedDataView,
     SeqData,
     SeqDataView,
+    SequenceCollection,
+    _names_list_tuple,
+    make_unaligned_seqs,
     seq_index,
```

Comment:
Change all imports to use the new versions of the MolType sequence and alphabet modules.